avaframe/in2Trans/transfromFields.py

```diff
@@ -0,0 +1,112 @@
+"""
+    ASCII file reader and handler
+
+"""
+
+import numpy as np
+import logging
+import pathlib
+
+# local imports
+import avaframe.com1DFA.DFAtools as DFAtls
+from avaframe.in3Utils import geoTrans as gT
+import avaframe.in2Trans.ascUtils as IOf
+
+# create local logger
+log = logging.getLogger(__name__)
+
+
+def convertDepthToThickness(depthDict, demDict, num=6):
```

Comment:
Mh, can we somehow get rid of the num parameter? Especially since it is different than the default from the com1 ini?
Ideas: set this to a default in the original function, or read the ini, or...?